package.json

```diff
@@ -49,6 +49,7 @@
     "@babel/runtime": "^7.0.0",
     "@types/geojson": "^7946.0.7",
     "@types/mapbox-gl": "^2.0.3",
+    "@types/viewport-mercator-project": "^6.1.1",
```

Comment:
The latest `viewport-mercator-project` is just an alias for `@math.gl/web-mercator`. The latter is already typed.

Reply:
@Pessimistress What do you mean by "just an alias"? `viewport-mercator-project` and `@math.gl/web-mercator` are still different packages. The latest `viewport-mercator-project` package doesn't match  to the latest `@math.gl/web-mercator` and doesn't contain type definitions. 

It makes sense to replace/migrate `viewport-mercator-project` with maintained `@math.gl/web-mercator`. But it doesn't match the goal of PR.

Reply:
This is the entire content of `viewport-mercator-project`: https://unpkg.com/browse/viewport-mercator-project@7.0.1/index.js

Reply:
@Pessimistress Ah, yes, my bad. It re-exports javascript, but doesn't re-export Typescript type definitions.

Reply:
@Pessimistress What do you think to replace `viewport-mercator-project` with `@math.gl/web-mercator`?

Reply:
Because in that case there are not too many choices.

- Migrate to `@math.gl/web-mercator`.
- Use `@types/viewport-mercator-project`.

Reply:
Choice #3 Make a PR on math.gl and re-export the types in viewport-mercator-project and we will publish a new release. These are all frameworks in the same vis.gl "family".

While that should be done regardless, perhaps this could indeed be a good time to stop importing the deprecated `viewport-mercator-project` module.

Reply:
There are legacy apps that import `WebMercatorViewport` from `viewport-mercator-project` per our old documentation. We should not remove a dependency in a patch release.

Reply:
https://github.com/uber-web/math.gl/pull/198

Reply:
Published `viewport-mercator-project@7.0.2` with TypeScript re-export.

Reply:
Removed `@types/viewport-mercator-project` and incremented `viewport-mercator-project` v7 version